src/pages/Winners2021.js

```diff
@@ -0,0 +1,369 @@
+                    </Link>
+                </figure>
+                <figure>
+                    <Link href="https://www.linkedin.com/in/ryomah/">
+                        <Headshot src={Ryoma} alt="Ryoma Harris"/>
+                        <Name>Ryoma Harris</Name>
+                    </Link>
+                </figure>
+            </Columns>
+
+            <Columns>
+                <ColumnText css={`padding-right: 2vw`}>
+                    <b><Subtitle fontSize="0.6rem">What was competing in the PCV Prize like?</Subtitle></b>
+                    <Text>"As a team, we're really glad to participate in the movement 
+                        to bring more environmental innovation to campus. Speaking to other 
+                        innovators and funders in the sustainability space was a rewarding 
+                        experience, especially since our entire team was brand new to 
+                        environmental entrepreneurship."</Text>
+                </ColumnText>
+                <ColumnText css={`padding-left: 2vw`}>
+                    <b><Subtitle fontSize="0.6rem">Future Plans?</Subtitle></b>
+                    <Text>"Over the summer, Folia is looking for web developers and other 
+                        help to build out our online platform and get a pilot program running 
+                        by the fall. We'd encourage all like-minded, passionate students to 
+                        reach out!"<br/>
+                        <br/>
+                        <b>View their deck!</b><br/>
+                        <Link href="https://docs.google.com/presentation/d/1qvGtOdH6_qirfyPLq628x1t8YId_kYMFndaaecIpHSA/edit?usp=sharing">
+                        https://docs.google.com/presentation/d/1qvGtOdH6<br/>_qirfyPLq628x1t8YId_kYMFndaaecIpHSA/edit?usp=sharing</Link><br/>
+                    </Text>
+                </ColumnText>
+            </Columns>       
+        </div>
+    )
+}
+
+const ThirdPlace = () => {
+    return (
+        <div>
+            <Subtitle css={`margin-top: 2vw`}>Third Place - Team Chaeto</Subtitle>
+
+            <Columns css={`text-align: center`}>
+                <img src={TeamChaeto} alt="Team Chaeto" css={`height: 25vw`}/>
+                <figure>
+                    <img src={ChrisNick} alt="Christopher and Nicholas Kwok" css={`height: 25vw`}/>
+                    <Name>Christopher and Nicholas Kwok</Name>
+                </figure>
+            </Columns>
+
+            
+            <Text>Team Chaeto pioneers an ecological, sustainable, and feasible solution 
+                to the existential crisis of ocean acidification. By modifying the activity 
+                of hydrogenase and carbonic anhydrase enzymes in chaetomorpha, the team 
+                invented chaeto biofilters to implement their novel strain of algae in 
+                marine ecosystems.</Text>
+
+            <Columns>
+                <ColumnText css={`padding-right: 2vw`}>
```

Comment:
prefer adding left and right props that, when truthy, apply this style in the styled-components styling above.